light-verifier-sdk/src/light_transaction.rs

```diff
@@ -578,24 +606,34 @@ impl<
         Ok(())
     }
 
+    fn validate_transaction_merkle_tree(
+        &self,
+        transaction_merkle_tree: &AccountInfo,
+    ) -> Result<()> {
+        let index = self
+            .input
+            .accounts
+            .unwrap()
+            .merkle_tree_authority
+            .transaction_merkle_tree_index;
```

Comment:
I don't like that we pass an additional account here, even if we take it from the lookup table because lookups are quite heap memory intensive.
We could modify the merkle tree account with a boolean which says it's the latest Merkle tree and set the boolean to false once we create a new tree.

Reply:
>We could modify the merkle tree account with a boolean which says it's the latest Merkle tree and set the boolean to false once we create a new tree.

That would require passing the old merkle tree manually as an input to the `initialize_merkle_tree_method`, so the program could set the boolean to `false`. So marking old Merkle trees as not the newest would be at will of SDK/client. Not sure if that's a good solution.

Reply:
nvm, fixed